src/Symfony/Component/Notifier/Bridge/Twilio/TwilioOptions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Twilio;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+final class TwilioOptions implements MessageOptionsInterface
```

Comment:
TO make this work, we need https://github.com/symfony/symfony/pull/48503 to be merged :exclamation:

Reply:
We merged #48503 ✅